vmware_aria_operations_integration_sdk/mp_build.py

```diff
@@ -147,15 +146,22 @@ def get_docker_registry(
     **kwargs: Any,
 ) -> str:
     docker_registry = get_config_value("docker_registry", config_file=config_file)
-    default_registry_value = f"harbor-repo.vmware.com/vmware_aria_operations_integration_sdk_mps/{adapter_kind_key.lower()}"
+    default_registry_value = get_config_value("default_container_registry")
```

Comment:
Do we ever set this anywhere or explain what it is? Does it make sense to have a global default?